components/ai_chat/resources/page/components/markdown_renderer/style.module.scss

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at https://mozilla.org/MPL/2.0/.
+
+.markdownContainer {
+
+  h1 {
+    font: var(--leo-font-heading-h1);
+  }
+
+  h2 {
+    font: var(--leo-font-heading-h2);
+  }
+
+  h3 {
+    font: var(--leo-font-heading-h3);
+  }
+
+  h4 {
+    font: var(--leo-font-heading-h4);
+  }
```

Comment:
it will fallback to browser styles:
![image](https://github.com/brave/brave-core/assets/8665427/0db92dc4-7ec9-4968-b908-8a8ad757b34f)

either we leave h5 and h6 stylings alone and let it fallback on browser styles or tell the renderer to not parse these elements

![image](https://github.com/brave/brave-core/assets/8665427/7bd09bc6-61fd-4159-af05-feccbcc09297)


cc @aguscruiz suggests to not parse these

Reply:
removed h5 and h6 parsing from the list